src/components/Input/Select.md

```diff
@@ -1,4 +1,10 @@
-### Default Style
+The select element creates a dropdown that allows users to choose one item from a list.
```

Comment:
Not sure if anything prevents using this for a multi-select. I didn't think to try.